client/webserver/site/src/js/app.js

```diff
@@ -154,14 +155,14 @@ export default class Application {
     pg.loader.style.backgroundColor = 'rgba(0, 0, 0, 0.5)'
     Doc.show(pg.loader)
     var hide
-    hide = e => {
-      if (!Doc.mouseInElement(e, pg.noteBox)) {
-        pg.noteBox.style.display = 'none'
-        unbind(document, hide)
+    hide = (el, e) => {
+      if (!Doc.mouseInElement(e, el)) {
+        el.style.display = 'none'
+        unbind(document, hide.bind(this, el))
```

Comment:
Does this work? I though you needed to refer to the same function object in `unbind -> removeEventListener`. `Function.bind` creates a new function object every time, no?